nebula/ui/components/VPNFocusOutline.qml

```diff
@@ -9,7 +9,7 @@ import Mozilla.VPN 1.0
 Rectangle {
     property variant focusedComponent
     property var focusColorScheme: VPNTheme.theme.blueButton
-    property var setMargins: -3
+    property real setMargins: -3
```

Comment:
I think we should use `int` in places where we realistically expect whole number, if for anything, for the sake of clarity so we know it would be unusual to set a decimal here